addons/web/static/tests/views/fields/numeric_fields_tests.js

```diff
@@ -180,4 +183,124 @@ QUnit.module("Fields", (hooks) => {
             assert.strictEqual(progressbarInputs[1].value, "0🇧🇪44🇧🇪🇧🇪");
         }
     );
+
+    QUnit.test(
+        "Numeric fields: NumpadDecimal key is different from the decimalPoint",
+        async function (assert) {
+            registry.category("services").remove("localization");
```

Comment:
nitpicking: you can use `force: true` when adding the fake service below.
or maybe it should work to simply patchWithCleanup the localization object from `@web/core/l10n/localization`

no need to change this, just saying to share knowledge ;-)

Reply:
I'll try patchWithCleanup. I kinda like it too.